src/http-launch.c

```diff
@@ -303,13 +303,19 @@ on_message (GstBus * bus, GstMessage * message, gpointer user_data)
       break;
     }
     case GST_MESSAGE_WARNING:{
-      gchar *debug;
-      GError *err;
+      gchar *debug = NULL;
+      GError *err = NULL;
 
       gst_message_parse_error (message, &err, &debug);
-      g_print ("Warning %s\n", err->message);
-      g_error_free (err);
-      g_free (debug);
+      //g_print ("Warning %s\n", err->message);
```

Comment:
Indeed this seemed weird to me. I thought I don't know enough about GStreamer.
Here are my command and logs: https://gist.github.com/singalen/dd9005709b41a16bafd0
Ubuntu 14.10 x86_64, GStreamer 1.4.3.